src/main/java/mate/academy/bookstore/service/impl/UserServiceImpl.java

```diff
@@ -31,7 +40,7 @@ public UserResponseDto register(UserRegistrationRequestDto requestDto)
         }
         User user = userMapper.toModel(requestDto);
         user.setPassword(encoder.encode(user.getPassword()));
-        user.getRoles().add(roleRepository.getByName(Role.RoleName.USER));
+        user.setRoles(Set.of(roleRepository.getReferenceById(userRoleId)));
```

Comment:
I believe it is redundant to make call to db here, if you want to make less db interactions on registration it makes sense to fetch Role on startup and set it to all without additional repo call